src/components/Scatterplot/index.js

```diff
@@ -72,12 +75,42 @@ const ScatterplotComponent = ({
     height,
   };
 
+  const { collectionIds, visibleCollections } = collections.reduce(
+    (info, c) => {
+      let { visibleCollections: updatedVisibleCollections } = info;
+      if (!c.hidden && c.yAxisDisplayMode !== AxisDisplayMode.NONE) {
+        updatedVisibleCollections += 1;
+      }
+      return {
+        collectionIds: {
+          ...info.collectionIds,
+          [c.id]: true,
+        },
+        updatedVisibleCollections,
```

Comment:
It's so that I can check if a series is specifying a valid collection.

I guess there are corner cases that make this insufficient; let me revisit this.

Reply:
Much cleaner, ready to be merged